docs/config/README.md

```diff
@@ -2342,6 +2342,44 @@ The module will be shown if any of the following conditions are met:
 format = "via [🏎  $version](red bold)"
 ```
 
+## Task Warrior
+The `Task Warrior` module shows the current started task in the Task Warrior and a symbol indicating if there are due tasks.
```

Comment:
The name here should be the actual string used in `starship.toml`, which should be `task_warrior`, if I'm not mistaken.